library/lib.sh

```diff
@@ -132,8 +142,8 @@ rolesEnableCallbackPlugins() {
     # Enable callback plugins for prettier ansible output
     callback_path=ansible_collections/ansible/posix/plugins/callback
     if [ ! -f "$collection_path"/"$callback_path"/debug.py ] || [ ! -f "$collection_path"/"$callback_path"/profile_tasks.py ]; then
-        ansible_posix=$(mktemp --directory)
-        rlRun "ansible-galaxy collection install ansible.posix -p $ansible_posix -vv"
+        ansible_posix=$(mktemp --directory -t ansible_posix-XXX)
+        rlRun "ansible-galaxy collection install ansible.posix -p $ansible_posix -vv --force"
```

Comment:
The use of `--force` is also tricky, because older versions of `ansible-galaxy` do not support it.  This is how runqemu.py does it - https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/test_scripts/runqemu.py#L96 - basically, run `ansible-galaxy collection install --help` and see if there is the option.  There is also a newer option which is better to use if available:

```
  --force-with-deps     Force overwriting an existing collection and its
                        dependencies.
```

Reply:
Fixed too